GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java

```diff
@@ -431,6 +431,10 @@ private void generateLoadEvent()
                     animator.start();
                 }
             }
+            synchronized (mNumTextures)
+            {
+                mNumTextures = -1;
```

Comment:
Why -1 and not 0. What is the difference?

Reply:
Still curious to know - what is the meaning of -1 here?

Reply:
This prevents it from trying to load textures after the asset has been loaded. Theoretically it should never get to generateLoadEvent() unless mTextures is 0.  But it will come here again if it tries to load another texture for that asset. It is basically a guard against having the asset generate a load event twice.

Reply:
@spodila1 please include Nola's note above as comments for this line in your next patch.  thanks.

Reply:
I have included.